components/micro-gateway-cli/src/main/resources/templates/basicAuthHandler.mustache

```diff
@@ -1,8 +1,8 @@
 {{#securityConfig}}{{#equals securityConfig.type "basic"}}
-auth:OutboundBasicAuthProvider {{endpointUrlType}}_{{operationId}}OutboundBasicAuthProvider = new({
+auth:OutboundBasicAuthProvider {{endpointUrlType}}_{{qualifiedServiceName}}{{operationId}}OutboundBasicAuthProvider = new({
     username: gateway:retrieveConfig("{{name}}_{{endpointUrlType}}_basic_username", "{{securityConfig.username}}"),
     password: gateway:retrieveConfig("{{name}}_{{endpointUrlType}}_basic_password", "")
 });
 
-http:BasicAuthHandler {{operationId}}_{{endpointUrlType}}OutboundBasicAuthHandler = new({{endpointUrlType}}_{{operationId}}OutboundBasicAuthProvider);
+http:BasicAuthHandler {{qualifiedServiceName}}{{operationId}}_{{endpointUrlType}}OutboundBasicAuthHandler = new({{endpointUrlType}}_{{qualifiedServiceName}}{{operationId}}OutboundBasicAuthProvider);
 {{/equals}}{{/securityConfig}}
```

Comment:
```suggestion
{{/equals}}{{/securityConfig}}

```

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/1200/commits/b34bf78c712b9c1122f77b28a6fbe3f26f1d37bc